src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -625,4 +626,28 @@ describe('Reimbursement Requests', () => {
       expect(newReimbursement).toStrictEqual(reimbursementTransformer(reimbursementMock));
     });
   });
+
+  describe('Edit Vendor Tests', () => {
+    test('Throws error if user isnt an admin', async () => {
+      await expect(
+        ReimbursementRequestService.editVendors('I Love Benny', GiveMeMyMoney.vendorId, wonderwoman)
+      ).rejects.toThrow(new AccessDeniedAdminOnlyException('only Admins can edit vendors'));
+    });
+
+    test('Vendor Name already exists', async () => {
```

Comment:
Small thing, but let's change this test name to 'Throws error if the vendor name already exists', so we can have consistent test names